Sources/PointFree/Emails/WelcomeEmails.swift

```diff
@@ -0,0 +1,271 @@
+import Dependencies
+import Models
+import IssueReporting
+import PointFreeRouter
+import StyleguideV2
+
+struct WelcomeEmail<Content: HTML>: EmailDocument {
```

Comment:
This is a general "layout" for welcome emails, and could probably be generalized for all emails.